src/main/java/teammates/common/util/Utils.java

```diff
@@ -33,5 +35,17 @@ public static Gson getTeammatesGson() {
                 .setPrettyPrinting()
                 .create();
     }
-
+    
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
@JosephineKwa Any chance this can take a `List<String>` instead? In which case it should be part of `StringHelper`.